### ISSUE TITLE
Review Naming Of Narrative User Input Fields

### ISSUE THREAD

**Issue by wendyrogers** [younginnovations/aidstream-old#98]
There is no real user need to contain the narratives in their own "Narrative" heading - this is just an XML thing - it may confuse users, who might think it means "lots of words" (as in narrative / story!)there is no real user need to contain the narratives in their own "Narrative" heading - this is just an XML thing - it may confuse users, who might think it means "lots of words" (as in narrative / story!)

**Comment by anjesh** [comment]
True. This is more of UI issue and the current version of Aidstream leverages the IATI XML structure to auto-generate form and hence it will require lots of low-level changes to AidStream to achieve this. However this issue will be taken care of in the next iteration of AidStream, which will also see drastic change in its UI.
